internal/k8s/fake_client.go

```diff
@@ -491,26 +491,60 @@ func (pf FakePortForwarder) ForwardPorts() error {
 }
 
 type FakePortForwardClient struct {
-	CreatePortForwardCallCount int
-	LastForwardPortPodID       PodID
-	LastForwardPortRemotePort  int
-	LastForwardPortHost        string
-	LastForwarder              FakePortForwarder
-	LastForwardContext         context.Context
+	mu               sync.Mutex
```

Comment:
I'd probably make `PortForwardCalls` unexported + provide a helper method that takes the lock and returns a copy of them (thankfully `FakePortForwarder` looks safe to copy, so doesn't need its own locks as well 😅). The `CreatePortForwardCallCount` + `LastXxx` helpers also don't currently use the mutex before reads.

Reply:
good call, done!